asv_bench/benchmarks/pandas/arithmetic.py

```diff
@@ -0,0 +1,379 @@
+import operator
+import warnings
+
+import numpy as np
+
+import modin.pandas as pd
+from modin.pandas import (
+    DataFrame,
+    Series,
+    date_range,
+)
+
+from .pandas_vb_common import numeric_dtypes
+
+import pandas.tseries
+from pandas.tseries import offsets
+try:
+    import pandas.tseries.holiday
+except ImportError:
+    pass
+
+
+class IntFrameWithScalar:
+    params = [
+        [np.float64, np.int64],
+        [2, 3.0, np.int32(4), np.float64(5)],
+        [
+            operator.add,
+            operator.sub,
+            operator.mul,
+            operator.truediv,
+            operator.floordiv,
+            operator.pow,
+            operator.mod,
+            operator.eq,
+            operator.ne,
+            operator.gt,
+            operator.ge,
+            operator.lt,
+            operator.le,
```

Comment:
```suggestion
            operator.mul,
```
I think we can leave 1 or 2 operations because they go through the same code path.